Tests/NIOSSLTests/NIOSSLIntegrationTest.swift

```diff
@@ -1633,10 +1663,8 @@ class NIOSSLIntegrationTest: XCTestCase {
         do {
             try clientChannel.writeInbound(buffer)
             XCTFail("Did not error")
-        } catch NIOSSLError.shutdownFailed {
-            // expected
         } catch {
-            XCTFail("Unexpected error: \(error)")
+            XCTAssertEqual(error.localizedDescription, "The operation couldn’t be completed. (NIOSSL.BoringSSLError error 0.)")
```

Comment:
I think you'll get a better description if you use `.description` instead of `.localizedDescription`.

Reply:
I addressed this in the latest commit.  Let me know if this is what you had in mind?